install_config/certificate_customization.adoc

```diff
@@ -19,82 +19,277 @@ This can be done during an
+`*openshift_master_overwrite_named_certificates*` parameters, which are
 configurable in the inventory file. More details are available about
 xref:../install_config/install/advanced_install.adoc#advanced-install-custom-certificates[configuring custom certificates with Ansible].
 
-.Example Custom Certificate Configuration with Ansible
-----
-# Configure custom named certificates
-# NOTE: openshift_master_named_certificates is cached on masters and is an
-# additive fact, meaning that each run with a different set of certificates
-# will add the newly provided certificates to the cached set of certificates.
-#
-# An optional CA may be specified for each named certificate. CAs will
-# be added to the OpenShift CA bundle which allows for the named
-# certificate to be served for internal cluster communication.
-#
-# If you would like openshift_master_named_certificates to be overwritten with
-# the provided value, specify openshift_master_overwrite_named_certificates.
+.Custom Certificate Configuration Parameters
+[source,yaml]
+----
+openshift_master_overwrite_named_certificates=true <1>
+openshift_master_named_certificates=[{"certfile": "/path/on/host/to/crt-file", "keyfile": "/path/on/host/to/key-file", "names": ["public-master-host.com"], "cafile": "/path/on/host/to/ca-file"}] <2>
+openshift_hosted_router_certificate=[{"certfile": "/path/on/host/to/app-crt-file", "keyfile": "/path/on/host/to/app-key-file", "cafile": "/path/on/host/to/app-ca-file"}] <3>
+----
+
+<1> To override `openshift_master_named_certificates` with
+the provided value, specify `openshift_master_overwrite_named_certificates=true`.
+<2> Provisions a xref:configuring-custom-certificates-master[master API certificate]. 
+<3> Provisions a xref:configuring-custom-certificates-wildcard[wildcard API certificate].
+
+Example parameters for a master API certificate:
+----
 openshift_master_overwrite_named_certificates=true
-#
-# Provide local certificate paths which will be deployed to masters
-openshift_master_named_certificates=[{"certfile": "/path/on/host/to/custom1.crt", "keyfile": "/path/on/host/to/custom1.key", "cafile": "/path/on/host/to/custom-ca1.crt"}]
-#
-# Detected names may be overridden by specifying the "names" key
-#openshift_master_named_certificates=[{"certfile": "/path/on/host/to/custom1.crt", "keyfile": "/path/on/host/to/custom1.key", "names": ["public-master-host.com"], "cafile": "/path/on/host/to/custom-ca1.crt"}]
+openshift_master_named_certificates=[{"names": ["master.148.251.233.173.nip.io"], "certfile": "/home/cloud-user/master-bundle.cert.pem", "keyfile": "/home/cloud-user/master.148.251.233.173.nip.io.key.pem" ]
+----
+
+Example parameters for a wildcard API certificate:
+----
+openshift_hosted_router_certificate={"certfile": "/home/cloud-user/star-apps.148.251.233.173.nip.io.cert.pem", "keyfile": "/home/cloud-user/star-apps.148.251.233.173.nip.io.key.pem", "cafile": "/home/cloud-user/ca-chain.cert.pem"}
 ----
 
 [[configuring-custom-certificates]]
```

Comment:
Not sure this PR is targeted to which OCP version, for ocp-3.9, just as I comment in previous doc PR before, "assetConfig" field is removed from master config yaml

Reply:
@gpei I removed `assetConfig` and replaced with `servingInfo`. PTAL particularly the sample code under _.Custom Certificates Configuration_